x-pack/plugins/security_solution/public/common/components/event_details/cti_details/threat_details_view.tsx

```diff
@@ -125,31 +107,108 @@ const buildThreatDetailsItems = (enrichment: CtiEnrichment) =>
       };
     });
 
+const renderAccordion = (
+  enrichment: CtiEnrichment,
+  index: number,
+  enrichments: CtiEnrichment[]
+) => {
+  const { id, field, provider, type, value } = getEnrichmentIdentifiers(enrichment);
+  const accordionTitle = `${field} ${value} ${provider ? i18n.PROVIDER_PREPOSITION : ''} ${
```

Comment:
The provider portion here could be its own component.

Reply:
added EnrichmentButtonContent component